doc/contributing/test.rst

```diff
@@ -133,7 +133,8 @@ drop and reinitialize the database before the run you can use the ``reset-db``
 option::
 
      nosetests --ckan --reset-db --with-pylons=test-core.ini ckan
-
+     
+You can find details on how to run a subset of tests in the `Nose Documentation <https://nose.readthedocs.io/en/latest/usage.html#selecting-tests>`_.
```

Comment:
Could you please format this line, because it don't pass tests currently

Reply:
I'm not particularly familiar with .rst.
Are there any resources on running the tests I can look at so I can fix this issue @smotornyuk ?
Thanks!

Reply:
https://docs.ckan.org/en/2.8/contributing/test.html - here's `How to` for CKAN testing.